lib/modules/manager/argocd/__fixtures__/validApplication.yml

```diff
@@ -129,3 +129,14 @@ spec:
         valueFiles:
           - $foo/values.yaml
 
+---
```

Comment:
can you add it to an existing application spec?

Reply:
Done.